static/js/dashboard/settings.js

```
@@ -0,0 +1,744 @@
+        const unconnectedProviders = availableProviders.filter(
+            provider => !connectedProviders.includes(provider.name)
+        );
+
+        if (unconnectedProviders.length > 0) {
+            // Show the Add Provider section
+            addProviderSection.style.display = 'flex';
+
+            unconnectedProviders.forEach(provider => {
+                const button = document.createElement('button');
+                button.className = 'btn btn-sm btn-outline';
+                button.onclick = () => eval(provider.action);
```

Biome (2.4.10)

[error] eval() exposes to security risks and performance issues.

(lint/security/noGlobalEval)